eva_submission/xlsx/xlsx_validation.py

```diff
@@ -98,7 +98,9 @@ def semantic_validation(self):
         taxid_and_species_list = set([(row['Tax Id'], row['Scientific Name']) for row in self.metadata['Sample'] if row['Tax Id']])
         for taxid, species in taxid_and_species_list:
             try:
-                scientific_name = get_scientific_name_from_ensembl(int(taxid))
+                scientific_name = get_scientific_name_from_taxonomy(int(taxid),
```

Comment:
Given the possible latency across the different stages (metadata validation, brokering etc.,) I think we should nail down the scientific name just once at the outset (we can decide when this should be #devmeeting), add it to the EVA and use the metadata from EVA from there on.